packages/entity-browser/src/components/EntityBrowser/InputFactory/InputFactory.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import TextFieldInputHandlerContainer from './../../../containers/TextFieldInputHandlerContainer'
+
+class InputFactory extends React.Component {
+
+  showField = () => {
+    return this.props.fieldDefinition.displayType !== 'HIDDEN'
+  }
+
+  getTypeMap = () => ({
+    'ch.tocco.nice2.model.form.components.simple.TextField': TextFieldInputHandlerContainer
```

Comment:
Create a container for FieldInputHandler seams a bit inconvenient for me. why no keep it a simple component and pass the properties down?